From 1907286f928dfe453096c796808f88899ffeb000 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Wed, 10 Jul 2024 09:44:41 +0300
Subject: [PATCH] [FSW-6580] fs1: move lfs disk to spsram base

with this patch, moved lfs disk to spsram base
instead of spsram+32kb, spsram base to 32kb
of spsram will be occupied by lfs disk.

Change-Id: Ife61106dee388aab86cec1d01ee2a8f72cee0ea6
---

diff --git a/arch/fs1/arc/dev_boot_info.h b/arch/fs1/arc/dev_boot_info.h
index cbfe35b..dbcae2e 100644
--- a/arch/fs1/arc/dev_boot_info.h
+++ b/arch/fs1/arc/dev_boot_info.h
@@ -9,9 +9,8 @@
 #include <soc.h>
 #include <boot_rom.h>
 
-/* add LFS offset */
 #define OFFSET_32KB		(32 * 1024)
-#define CONFIG_BOOTINFO_ADDR	(HL_FW_MEM_BASE + HL_SCRATCHPAD_SRAM_OFFSET + OFFSET_32KB)
+#define CONFIG_BOOTINFO_ADDR	(HL_FW_MEM_BASE + HL_SCRATCHPAD_SRAM_OFFSET)
 #define CONFIG_BOOTINFO_SIZE	OFFSET_32KB
 
 #endif	/* _DEV_BOOT_INFO_H_ */
